Dockerfile

```diff
@@ -17,6 +17,7 @@ EXPOSE 80
 WORKDIR /usr/share/nginx/html
 COPY ./env.sh .
 COPY ./config.template.json .
+COPY ./.env .
```

Comment:
This wasn't in the build for a reason. You need to pass in the correct ENV variables when booting the application for it to run.

Reply:
ok